From 100d7feb1d7d933eb4959f879a6a516dba9eb427 Mon Sep 17 00:00:00 2001
From: Dennis Wenzel <b21659@freescale.com>
Date: Thu, 14 Jul 2011 21:38:33 -0500
Subject: [PATCH 228/505] ENGR00153156: build error in imx-test for
 mxc_epdc_x11_fb_test

Added missing XRandR library include to the link.

Signed-off-by: Dennis Wenzel <b21659@freescale.com>
---
 test/mxc_x11_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_x11_test/Makefile b/test/mxc_x11_test/Makefile
index f037ad4..dd4ca7c 100644
--- a/test/mxc_x11_test/Makefile
+++ b/test/mxc_x11_test/Makefile
@@ -1,7 +1,7 @@
 # list of platforms which did not want this test case
 EXCLUDE_LIST:=
 CFLAGS += -lm
-LIBS += -lX11 -lXdamage -lXrender
+LIBS += -lX11 -lXdamage -lXrender -lXrandr
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/mxc_epdc_x11_fb_test.out $(OBJDIR)/x11_overlay.out
-- 
1.8.0

